### DISCUSSION TITLE
cmd://{FIREFOX} works in Keepass/Windows, but not KeepassXC/Linux Mint?

### DISCUSSION THREAD

**Discussion by flagellatedcucumber** [keepassxreboot/keepassxc#6319] [Q&A] [answered]
In Keepass/Windows, a URL = "cmd://{FIREFOX} ""https://www.whatevva.com"" " works fine.

In KeepassXC v.2.4.3 in Linux Mint, the same URL seems to be recognised by KeepassXC - it produces a dialogue box to confirm whether the command should always be done for this entry - but once the user clicks on OK, then nothing happens.  In this case, the placeholder {FIREFOX} doesn't launch Firefox.

The Linux Mint environment has Chromium as the default browser (for now).

What else do I need to know, or to find out (and how?) to diagnose the issue?

**Comment by droidmonkey** [comment, marked as answer]
Just use cmd://firefox

**Comment by flagellatedcucumber** [comment]
Thanks - that works.
